torch/csrc/jit/codegen/cuda/type.cpp

```diff
@@ -464,6 +501,22 @@ std::string stringifyThread(const ParallelType ptype) {
   return parallel_type2string(ptype);
 }
 
+std::string typePrefix(const DataType data_type) {
+  switch (data_type) {
+    case DataType::Bool:
+      return "b";
+    case DataType::Double:
+      return "d";
+    case DataType::Float:
```

Comment:
Is using `f` for half type intended? Is it because we do not do any internal computation on half?